config.yml

```diff
@@ -1 +0,0 @@
-datastore-class: xcube_geodb_openeo.core.GeoDBDataStore
```

Comment:
Rename to `datastore_class`, so it is not impossible to deserialize configuration into Python objects.